who-we-are.md

```diff
@@ -32,26 +28,45 @@ We promote a company culture that appreciates collaborative professional growth.
 
 We typically work on smaller 2-6 month projects that use forward-leaning technologies. This allows our team to gain experience with the latest technologies in different industries. For projects that have longer durations we attempt to rotate team members so that the work does not feel stale, and so that more of the team has experience with any given project.
 
-In addition we also provide access to online training opportunities, like CodeSchool.com, we sponsor employee attendance at various technical conferences, and we encourage community involvement through technical meetups and open source contributions.
-
-Career growth is facilitated and tracked via the Buddy System. Individuals meet routinely with a buddy of their choice (not all team members are buddies, but every team member has a buddy) to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a manager, and the Buddy System is no different: we do not review our teammates, we advocate for them. All Buddy System documents are transparent and [available on the shared Drive](https://b2io.slack.com/archives/C904HLD5L/p1516982662000944).
+In addition we also provide access to online training opportunities and sponsor employee attendance at various technical conferences. We encourage community involvement through technical meetups and open source contributions.
 
+Career growth is facilitated and tracked. Individuals meet routinely with their supervisor to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a direct manager. we do not review our teammates, We advocate for them.
```

Comment:
It was Drew as part of the review process the last time, but conceivably it could be some other person at some point so just simplify to `team member`?
```suggestion
Career growth is facilitated and tracked. Individuals meet routinely with a team member to discuss career goals, track goal progress, and discuss feedback. Our flat hierarchy means that no one has a direct manager. We are accountable to, and advocate for each other.
```